compare.js

```diff
@@ -12,34 +12,42 @@ const root = await protobuf.load(path.join(process.cwd(), "sitesurvey.proto"));
 
 const siteSurvey = root.lookupType("asset_tracker_v2.WiFiSiteSurvey");
 
+// Eliminate data that is not correct (missing some info)
+const data = jsonData.v.filter(({ mac, chan, ssid, rssi }) => {
+  return mac != undefined
+      && chan != undefined
+      && ssid != undefined
+      && rssi != undefined
+})
+
 // Create the protobuf message, convert Mac addresses to long to further save data
 // We don't have to use short keys (v, chan) for the protobuf message (as in the JSON)
 // and can use full, descriptive names.
 const message = siteSurvey.create({
-  ...jsonData,
```

Comment:
This removes the `timestamp`.

Reply:
My bad, I'm going to add that later, but the result shouldnt change much.